tools/api-markdown-documenter/biome.jsonc

```diff
@@ -0,0 +1,7 @@
+{
+	"$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+	"extends": ["../../biome.jsonc"],
+	"files": {
+		"ignore": ["**/test/test-data/**"]
```

Comment:
Does it affect things if we let it format the json files there? cc @Josmithr for comments.

Also, isn't it redundant with the change to the root biome config's `files.ignore`?

Reply:
It's not redundant unfortunately. Biome's base dir is the cwd, so when you invoke it here the root pattern doesn't match. We could use more general patterns in the root, but that risks ignoring stuff we don't want to.